cpp/cmake/IceConfig.cmake

```diff
@@ -23,28 +23,65 @@ if(NOT DEFINED Ice_ARCHITECTURE)
   endif()
 endif()
 
+if(EXISTS ${Ice_HOME}/cpp)
+  set(Ice_SOURCE_BUILD ON)
+endif()
+
+if(Ice_SOURCE_BUILD)
+  list(APPEND ice_include_path_suffixes "cpp/include")
+  list(APPEND ice_bin_path_suffixes "cpp/bin")
+  list(APPEND ice_lib_path_suffixes_release "cpp/lib")
+
+  if (Ice_ARCHITECTURE)
+    list(APPEND ice_lib_path_suffixes_release "cpp/lib/${Ice_ARCHITECTURE}")
+  endif()
+
+  if(WIN32)
+    list(APPEND ice_bin_path_suffixes "cpp/bin/${Ice_ARCHITECTURE}/Release" "cpp/bin/${Ice_ARCHITECTURE}/Debug")
```

Comment:
aren't there separate suffixes for Release and Debug?

Reply:
Not really for finding this binary.